app/build.gradle

```diff
@@ -34,6 +34,7 @@ android {
         useIR = true
     }
     buildFeatures {
+        viewBinding true
```

Comment:
I started to add the beginnings of one of the projects I saw you all were going to implement which is the AutoCompleteTextview. The link to the tutorial on how to add this in is here --> https://tutorialwing.com/android-autocompletetextview-using-kotlin-with-example/

Reply:
What purpose does this serve?
This looks like an incomplete feature.
Are you using these anywhere?  I don't see them used.
If not in use, remove them.  Unused code is clutter, and bad documentation, which increases technical debt.
Only complete features, which meet the team's Definition of Done, should be merged to Main/Master.

Reply:
@discospiff My idea behind this was to send a resource for the team to begin to look at and implement the start of that resource.